### PR TITLE
Operator attestation policy

### PR DESCRIPTION
**Pull request by turadg**
TODO in code

## Description
Update `TransactionFeed` to publish when a majority of operators to attest (previous required unanimous). Conflicts log a loud error and don't publish.

Also updates the contract test to have 3 operators (per Mainnet policy) and rely on unit tests for the other values.

### Security Considerations

Adds assertions to the deploy-config to help ensure that Mainnet has three unique operators, per security policy.

### Scaling Considerations
Each new attestation iterates over three stores for matching values.

### Documentation Considerations
Nothing end-user. OCW is documented elsewhere.

### Testing Considerations
CI

### Upgrade Considerations
not yet deployed.

contract is upgradable but changing the settlementAccountAddress will require creating a new TransactionFeed (as is the case for the Settler object.

### CODE REVIEW

File: packages/fast-usdc/test/exos/transaction-feed.test.ts
Comment (dckc) [inline packages/fast-usdc/test/exos/transaction-feed.test.ts:120]: good; the "unknown transaction" test mixed in testing this, and I wondered about reducing coverage.

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (dckc) [inline packages/fast-usdc/src/exos/transaction-feed.js:122]: for... what?

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (dckc) [inline packages/fast-usdc/src/exos/transaction-feed.js:161]: this seems to be the gist of it.

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (dckc) [inline packages/fast-usdc/src/exos/transaction-feed.js:187]: the change from `pending.values()` to `found`... I had to think a bit to convince myself that make no difference.

File: packages/fast-usdc/test/exos/transaction-feed.test.ts
Comment (dckc) [inline packages/fast-usdc/test/exos/transaction-feed.test.ts:62]: I haven't thought carefully about whether this is sufficient test coverage.

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (dckc) [inline packages/fast-usdc/src/exos/transaction-feed.js:173]: is this a red-alert situation? It seems like we recover: we don't act on inconsistent evidence.

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (dckc) [inline packages/fast-usdc/src/exos/transaction-feed.js:178]: if we have to use this in anger, I expect we'll want to see which operator submitted which evidence.

is that already traced?

File: packages/fast-usdc/test/exos/transaction-feed.test.ts
Comment (dckc) [inline packages/fast-usdc/test/exos/transaction-feed.test.ts:108]: why ask for updates since 0 again? isn't the n+1th call supposed to pass in the updateCount from the nth call?

File: packages/fast-usdc/test/exos/transaction-feed.test.ts
Comment (dckc) [inline packages/fast-usdc/test/exos/transaction-feed.test.ts:112]: ooh. tricky.

File: packages/fast-usdc/src/utils/deploy-config.js
Comment (dckc) [inline packages/fast-usdc/src/utils/deploy-config.js:162]: :+1:

**Review by dckc — APPROVED** [review]
looks pretty good

this review is a bit rushed, so you may want to get more eyes.
or let me know if you want me to look more closely.

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (turadg) [reply packages/fast-usdc/src/exos/transaction-feed.js:173]: I think it deserves immediate operational attention if two oracles are disagreeing. At least one is unreliable and may require immediate action.

File: packages/fast-usdc/test/exos/transaction-feed.test.ts
Comment (turadg) [reply packages/fast-usdc/test/exos/transaction-feed.test.ts:108]: It's just a handy way to ask for the latest without risk of a hanging await while writing tests

File: packages/fast-usdc/src/exos/transaction-feed.js
Comment (turadg) [reply packages/fast-usdc/src/exos/transaction-feed.js:122]: I must have gotten a Slack notification!

> The operatorKit is responsible for preventing a disabled kit from attesting.

I'll get that in before merge. I'll probably wait for another reviewer's comments before pushing again.

**Review by heavypackets — APPROVED** [review]

**Pull request merged**
